opencti-platform/opencti-graphql/src/modules/ingestion/ingestion-csv-domain.ts

```diff
@@ -93,6 +93,20 @@ export const ingestionCsvEditField = async (context: AuthContext, user: AuthUser
   return notify(BUS_TOPICS[ABSTRACT_INTERNAL_OBJECT].EDIT_TOPIC, element, user);
 };
 
+export const ingestionCsvResetState = async (context: AuthContext, user: AuthUser, ingestionId: string) => {
+  await patchCsvIngestion(context, user, ingestionId, { current_state_hash: '' });
+  const ingestionUpdated = await findById(context, user, ingestionId);
+  await publishUserAction({
+    user,
+    event_type: 'mutation',
+    event_scope: 'update',
+    event_access: 'administration',
+    message: `reset state of csv ingestion ${ingestionUpdated.name}`,
+    context_data: { id: ingestionId, entity_type: ENTITY_TYPE_INGESTION_CSV, input: ingestionUpdated }
+  });
+  return notify(BUS_TOPICS[ABSTRACT_INTERNAL_OBJECT].EDIT_TOPIC, ingestionUpdated, user);
+};
+
```

Comment:
Shouldn't we also reset the state of the csv ingestion when 'added_after_start' is modified in the ingestionCsvEditField ? The same way it is done in taxii feeds

Reply:
As far as I know added_after_start is totally useless on csv feed, to me it's a copy paste mistake from taxii feeds since it's not used at all in ingestionManager.

Reply:
You're right, I didn't dive that deep into the code so I thought it was useful in some way. 
Not related to your PR,  but when looking more at the code, I saw that the current_state_date is also completely useless now, is that intended? The user can still input an 'Import from date' date when creating a CSV feed, but since it is not used at all, maybe we should create a different issue to completely remove this field/input?

Reply:
I just remove from UI for now to avoid API breaking changes.

Reply:
I removed from UI because I think it's misleading, I haven't change the API to avoid breaking changes.